uber/site_sections/signups.py

```diff
@@ -67,10 +68,12 @@ def volunteer(self, session, id, csrf_token=None, requested_depts='', message='S
         }
 
     @check_shutdown
-    def shifts(self, session):
+    def shifts(self, session, tgtDate='', state=''):
```

Comment:
Minor style nitpick, but our convention is to use ``snake_case_variable_names`` in our Python code rather than ``camelCaseVariableNames``.